### PR TITLE
fix scripts, docs and CI

### PR DESCRIPTION
**Pull request by themanforfree**
Please briefly answer these questions:

* what problem are you trying to solve? (or if there's no problem, what's the motivation for this change?)
fix scripts, docs and CI after refactor args
* what changes does this pull request make?
fix bugs
* are there any non-obvious implications of these changes? (does it break compatibility with previous versions, etc)
No

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -5,16 +5,19 @@
...
 3. Download or build `etcdctl` from [etcd](https://github.com/etcd-io/etcd) project.
 4. Use `etcdctl` to operate the cluster:
     ```bash
     etcdctl --endpoints=http://127.0.0.1:2379 put foo bar
 
     etcdctl --endpoints=http://127.0.0.1:2379 get foo
-    ```
\ No newline at end of file
+    ```
+
```
Comment (markcty) [inline USAGE.md:21]: Blank lines?

**Review by markcty — CHANGES_REQUESTED** [review]

File: USAGE.md [hunk 2]
```diff
@@ -5,16 +5,19 @@
     ```bash
     # Run in 3 terminals. If you want more logs, add `RUST_LOG=debug` before the command.
 
-    ./xline --name node1 --cluster-peers 127.0.0.1:2380 127.0.0.1:2381 --self-ip-port 127.0.0.1:2379 --leader-ip-port 127.0.0.1:2379 
+    ./xline --name node1 --members node1=127.0.0.1:2379,node2=127.0.0.1:2380,node3=127.0.0.1:2381
```
Comment (Phoenix500526) [inline USAGE.md:8]: Is here missing the `--is-leader` option?

File: USAGE.md (hunk 1 above)
Comment (themanforfree) [reply USAGE.md]: removed

File: USAGE.md (hunk 2 above)
Comment (themanforfree) [reply USAGE.md:8]: we can  run without this flag now, curp will elect a leader

Comment (markcty) [reply USAGE.md:8]: I guess we still need it. Because as we discussed yesterday, curp won't start an election automatically when it starts.

Comment (Phoenix500526) [reply USAGE.md:8]: Apart from the reasons that markcty has mentioned, I think it's a good idea to add this option, as doing so can provide developers who are green to Xline with more details on how to use it. Just my two cents.

Comment (themanforfree) [reply USAGE.md]: added

**Review by Phoenix500526 — APPROVED** [review]

**Pull request merged**